src/components/FlameGraph/FlameGraph.jsx

```diff
@@ -50,15 +50,23 @@ class FlameGraph extends Component {
             'handleSearchClick',
             'handleOnKeyDown',
             'updateSearchQuery',
+            'handleLayoutChange',
         ].forEach((k) => {
           this[k] = this[k].bind(this);
         });
+
+        const preferredLayout = () => {
+            if(localStorage.getItem('layout')){
+                return localStorage.getItem('layout');
+            } else return 'flame';
```

Comment:
Might be better to keep the 'flame' and 'icicle' strings as constants exported in a config file. `src/config.jsx`